### PR TITLE
changes passive vent sprites to be more unique

### PR DESCRIPTION
**Pull request by SonixApache**
dreamseeker_2020-12-01_18-09-50
requested by spyroshark

🆑 
 - imageadd: Changed passive vent sprites to be more unique.

### CODE REVIEW

File: code/ATMOSPHERICS/components/unary/vent.dm [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /obj/machinery/atmospherics/unary/vent
 	icon = 'icons/obj/atmospherics/pipe_vent.dmi'
-	icon_state = "base"
-	name = "Vent"
-	desc = "A large air vent"
+	icon_state = "base_pas"
+	name = "Passive Vent"
```
Comment (DamianX) [inline code/ATMOSPHERICS/components/unary/vent.dm]:
```suggestion
	name = "passive vent"
```

Comment (SonixApache) [reply code/ATMOSPHERICS/components/unary/vent.dm]: Atmos things are Proper Nouns for some reason, just following the naming scheme.

Comment (DamianX) [reply code/ATMOSPHERICS/components/unary/vent.dm]: it's dum

Comment (SonixApache) [reply code/ATMOSPHERICS/components/unary/vent.dm]: change them all in a seperate PR, don't 🗞️ me for it

Comment (DamianX) [reply code/ATMOSPHERICS/components/unary/vent.dm]: I mean, you were changing the name unatomically, I figured you were cool with this. Whatever.

**Comment by cloutcode** [comment]
i just thought about this recently this proves the CIA is monitoring my thought process

**Pull request merged**